zhaquirks/sonoff/snzb06p.py

```diff
@@ -0,0 +1,88 @@
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify, Ota
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+SONOFF_CLUSTER_FC11_ID = 0xFC11
+SONOFF_CLUSTER_FC57_ID = 0xFC57
+ATTR_SONOFF_ILLUMINATION_STATUS = 0x2001
+
+
+class IlluminationStatus(t.enum8):
+    """Last measureed state of illumination enum."""
+
+    Dark = 0x00
+    Light = 0x01
+
+
+class SonoffFC11Cluster(CustomCluster):
+    """Sonoff manufacture specific cluster that provides illuminance."""
+
+    name = "Sonoff Manufacture Specific Cluster at 0xFC11"
+    cluster_id = SONOFF_CLUSTER_FC11_ID
+    ep_attribute = "sonoff_manufacturer_specific_FC11"
+    attributes = {
+        ATTR_SONOFF_ILLUMINATION_STATUS: ("last_illumination_state", IlluminationStatus)
```

Comment:
We could also add an `Illuminance` cluster extending `LocalDataCluster` and then update that cluster/attribute cache from this cluster using something like this: [zhaquirks/xiaomi/aqara/driver_curtain_e1.py#L56-L64](https://github.com/zigpy/zha-device-handlers/blob/a014ef5969045bf3e3be8664d2778c7a5ed8c2a5/zhaquirks/xiaomi/aqara/driver_curtain_e1.py#L56-L64)

Tbh, I'm not sure if it's better to have a custom entity in this case or fake the cluster and use the "default" one.

Reply:
Maybe I miss understand what you mean, but wouldn't that be a worse UX as the last illumination state from the device is only ever "light" and "dark" and we'd need to choose arbitrary values in lx for what those two states are?

Reply:
Yep, probably. Just thought about it. I'll have a closer look at your ZHA PR soon to see how you implemented it right now.